### DISCUSSION TITLE
problem with passing verification as a student

### DISCUSSION THREAD

**Discussion by masshitro** [community/community#65883] [GitHub Education]
### Hi everyone,

I am a student of a university located in Ukraine, and our corporate email has not been working for a year. However, only the corporate email is accepted for verification. How can I pass verification by providing my student ID and Gmail, for example?

**Comment by arafajs** [comment]
Unfortunately, I cannot guarantee that this will work, as it depends on the specific verification process being used. However, Here are some things you can try:

* Contact your university's IT department and ask them if there is a way to verify your student identity using your student ID and Gmail.
* If your university has a student portal, try logging in with your student ID and Gmail. This may trigger a verification process that allows you to pass using these credentials.
* If you have a copy of your student ID, try uploading it to the verification process.
* If you have a friend or family member who is also a student at your university, see if they can help you pass verification by providing their corporate email.
* Provide other information that can help verify your identity, such as your date of birth, home address, or phone number.
* Explain the situation to the person or organization that is requiring verification and ask if they can make an exception for you.
* If all else fails, you may need to create a new email account using a different provider.

**Comment by denverharry** [comment]
I am also facing the same issue and i also facing this issue on the [spartanburg concrete patio](https://www.spartanburgconcretesolutions.com/concrete-patios-spartanburg.html) website.
